client/clientservice/src/request_service.cpp

```diff
@@ -119,12 +119,16 @@ void RequestServiceCallData::sendToConcordClient() {
           break;
         case (static_cast<uint32_t>(bftEngine::OperationResult::CONFLICT_DETECTED)):
           LOG_INFO(logger, "Request failed with CONFLICT_DETECTED error for cid=" << req_config.correlation_id);
-          status = grpc::Status(grpc::StatusCode::ABORTED, "Aborted");
+          status = grpc::Status(grpc::StatusCode::ABORTED, "Aborted due to conflict detected");
```

Comment:
Please adapt the [API documentation](https://github.com/vmware/concord-bft/blob/master/client/proto/request/v1/request.proto#L31) to reflect this change.

Reply:
Updated comments for error message strings